server/src/main/java/com/lunchtime/models/Person.java

```diff
@@ -69,13 +69,6 @@
     @Column(name = "status", columnDefinition = "varchar(50) default 'ACTIVE'")
     private Status status;
 
-    @ManyToOne(fetch = FetchType.EAGER)
-    @JoinTable(name = "user_roles",
```

Comment:
You deleted user_roles join table from model class. But in database (dev) this table will still be there. I think without script (to remove this table in db) it will make error.

Reply:
I also think you need to delete the unused table.

Reply:
It doesn't make error on my environment. I agree, we should remofe it from scripts.

Thanks, fixed this yet. Now you can check again.